src/test/acceptance/features/confirmation-code-navigation.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Confirmation code navigation
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  Once I have entered my confirmation code
+  If I change the contact details used to receive that code
+  I have to go through the send and enter code part of the application to receive a new code.
```

Comment:
How about `I have to re-validate those contact details`?

Reply:
Reworded.